pages/en/lb3/Authentication-authorization-and-permissions.md

```diff
@@ -68,55 +68,87 @@ module.exports = function enableAuthentication(server) {
 
 ### Preparing access control models
 
-Make sure the `User` model, (and possibly the `AccessToken` model) is configured appropriately according to your set-up.
+You must configure the `User` model, (and possibly the `AccessToken` model) according to your set-up.
 
-Normally you should have already implemented at least one custom user model, extending the built-in `User` model, as described in [Using built-in models](Using-built-in-models.html#user-model).
+Normally you will implement at least one custom user model, extending the built-in `User` model, as described in [Using built-in models](Using-built-in-models.html#user-model).
 
-Usually you don't need to extend or customize the built-in models `Role`, `RoleMapping`, and `ACL`. Just make sure they are declared in the `model-config.json` configuration file, or in case you don't require either to customize the `AccessToken` model that you pass a datasource to the `enableAuth()` method as follows:
+Usually you don't need to extend or customize the built-in models `Role`, `RoleMapping`, and `ACL`. Just make sure they are declared in the `model-config.json` configuration file, or if you don't need to customize the `AccessToken` model, pass a datasource to the `enableAuth()` method as follows:
 
 ```javascript
 server.enableAuth({ datasource: 'db' });
 ```
 
-{% include note.html content="
-Passing a `datasource` to the `enableAuth()` method as shown here will let LoopBack take care of attaching any built-in models required by the access control feature, which is suitable for most applications.
+{% include note.html content="Passing a `datasource` to the `enableAuth()` method as shown here will let LoopBack take care of attaching any built-in models required by the access control feature, which is suitable for most applications.
 " %}
 
-{% include tip.html content="
-Whether you can use the built-in `AccessToken` model or create a custom accessToken model extending the built-in model depends on whether you plan to use one or several user models extending the built-in `User` model. Both cases are covered in the next two sections.
+{% include tip.html content="Whether you can use the built-in `AccessToken` model or create a custom model that extends `AccessToken` depends on whether you plan to use one or several user models extending the built-in `User` model, as described in the following sections.
 " %}
 
-#### Access control with a single user model
+#### Access control with a single user model and built-in AccessToken model
```

Comment:
I'll change this L4 heading to `Access control with a single user model` and then change the heading `Customizing the AccessToken model` to L5.